modules/ROOT/pages/7.6.0-release-notes.adoc

```diff
@@ -92,6 +92,17 @@ In previous versions of {productname}, the `Shift+Enter` key was not properly ha
 
 In {productname} {release-version}, mentions in comments now handle the `Shift+Enter` key by inserting the highlighted user into the comment text area and closing the dropdown, ensuring consistent functionality across the editor and comment areas.
 
+==== Editor Focus after Deleting a Comment
+// #TINY-11293
+
+Previously, an issue was identified where when a user deleted a comment, the focus exited the comments sidebar and returned to the editor instead of returning to the conversation card. This issue also occurred when a new comment was created inside an existing conversation, and when an entire new conversation was created.
```

Comment:
I wouldn't call this an issue, it's more of a change or improvement to the UX.

Reply:
@hamza0867 I've made some changes based on your suggestion.